pkg/storage/utils/eosfs/eosfs.go

```
@@ -2284,11 +2289,13 @@ func (fs *eosfs) convert(ctx context.Context, eosFileInfo *eosclient.FileInfo) (
 				},
 			},
 		},
-		ArbitraryMetadata: &provider.ArbitraryMetadata{
-			Metadata: filteredAttrs,
-		},
 	}
-
+	if eosFileInfo.Attrs[eosLockKey] != "" {
+		// populate the lock, fail silently if unable to decode
+		if l, _ := decodeLock(eosFileInfo.Attrs[lockPayloadKey], eosFileInfo.Attrs[eosLockKey]); l != nil {
+			info.Lock = l
```

An error log would be useful for this silent fail.